frontend/pages/household/mealplan/planner/edit.vue

```diff
@@ -347,6 +350,13 @@ export default defineComponent({
       newMeal.existing = false;
     }
 
+    function isCreateDisabled() {
+      if (dialog.note) {
+        return !newMeal.title.trim();
+      }
+      return !newMeal.recipeId;
+    };
+
```

Comment:
Pretty sure this needs to be converted to a computed property to handle reactivity correctly.